test/integration/scheduler_perf/dra/consumablecapacity/consumablecapacity_test.go

```diff
@@ -35,9 +36,21 @@ func TestMain(m *testing.M) {
 }
 
 func TestSchedulerPerf(t *testing.T) {
-	perf.RunIntegrationPerfScheduling(t, "performance-config.yaml")
+	// Verify correct behavior with all available allocators for this feature.
+	for _, allocatorName := range []string{"experimental"} {
```

Comment:
This list and the `structured.EnableAllocators` call below is something that we will have to update if we reorganize which allocator supports which features. If we don't update when e.g. "incubating" also supports consumable capacity, then nothing breaks, we just won't have the full (here) or right (below) test coverage.

I don't see a good way to prevent this automatically. Perhaps I need to cross-reference this here in the k8s.io/dynamic-resource-allocation/structured repo's comments or README?

Reply:
I added `staging/src/k8s.io/dynamic-resource-allocation/structured/internal/README.md` with a brain dump of our discussions around this.